### ISSUE TITLE
visual studio trouble

### ISSUE THREAD

**Issue by halogreenman** [Garethp/ScreepsAutocomplete#99]
im trying to get this to work with the visual studio IDE but i cant seem to find any of the buttons to press in the readme, but when i found something that said ASP.NET visual studio said there was no relevant results, i would really appreciate some help

**Comment by SirArthur2009** [comment]
Yes, I'm having same problem, I'm just using API rn to figure out commands
